src/reducers/error.js

```diff
@@ -0,0 +1,18 @@
+import * as t from '@/actions/types';
+
+const initialState = '';
+
+export default (state = initialState, { type, payload }) => {
+  if (type === t.SHOW_ERROR) {
+    return payload.error;
+
+  } else if (type === t.DISMISS_ERROR) {
```

Comment:
We can remove the `else` and make two distinct `if`-statements here – easier to read 😄